File: Tarea2_Dato_Abstracto_28-08-25/LEM_Dato_Abstracto.java
public class persona {
    String nombres;
    String ape1;
    String ape2;

    public static void main(String[] args) {
        persona p = new persona();
        p.nombres = "Luis Eduardo";
        p.ape1 = "Manzanarez";
        p.ape2 = "Ramos";

        System.out.println(p.nombres + " " + p.ape1 + " " + p.ape2);
    }
}

File: Tarea2_Dato_Abstracto_28-08-25/LEM_Dato_Abstracto.js
class persona {
  constructor(nombres, ape1, ape2) {
    this.nombres = nombres;
    this.ape1 = ape1;
    this.ape2 = ape2;
  }
}

// Uso
let p = new persona("Luis Eduardo", "Manzanarez", "Ramos");
console.log(p.nombres, p.ape1, p.ape2);